<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegionWizard</class>
 <widget class="QWizard" name="RegionWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>278</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>277</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>279</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Region Wizard</string>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ClassicStyle</enum>
  </property>
  <widget class="QWizardPage" name="page1_Region">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>531</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Region data</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>40</y>
       <width>491</width>
       <height>140</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="formLayoutRegionData">
      <item row="0" column="0">
       <widget class="QLabel" name="label_19">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="regionName">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="accessibleName">
         <string>region_name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QWidget" name="widget_2" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_22">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>UUID</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_12">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="accessibleName">
         <string>RegionUUID</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(238, 242, 255);</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnNewUUID">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>N&amp;ew UUID</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_23">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Location</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_13">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="accessibleName">
         <string>Location</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QWidget" name="widget_3" native="true"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_24">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Internal Port</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="accessibleName">
         <string>InternalPort</string>
        </property>
        <property name="inputMask">
         <string>000000; </string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="btnNextPort">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Ne&amp;xt port</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="externalHostNameLabel">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>External Host Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEditExternalHostName">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="accessibleName">
         <string>ExternalHostName</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QWidget" name="widget" native="true"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWizardPage" name="page2_master">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Master avatar</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>441</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayoutAvatarData">
      <item row="0" column="0">
       <widget class="QLabel" name="label_21">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>First Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="accessibleName">
         <string>MasterAvatarFirstName</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_25">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Last Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_10">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="accessibleName">
         <string>MasterAvatarLastName</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_26">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_11">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="accessibleName">
         <string>MasterAvatarSandboxPassword</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
